zsh/.zgen-setup

```diff
@@ -7,12 +7,14 @@
 # BSD licensed, see LICENSE.txt
 
 # Clone zgen if you haven't already
-if [[ ! -f ~/zgen/zgen.zsh ]]; then
-  pushd ~
+ZGEN_PARENT_DIR=$HOME/github
+if [[ ! -f $ZGEN_PARENT_DIR/zgen/zgen.zsh ]]; then
+  pushd $ZGEN_PARENT_DIR
```

Comment:
I would normally, but since this is a starter kit I want everything to be nice and easy to read for people who are new to using a shell. I originally wrote this to help out some interns at a previous `$JOB` and for some of them it was their first exposure to sh-like languages.